.gitignore

```diff
@@ -54,7 +54,6 @@ profile.cov
 /pkg/cmd/grafana-cli/grafana-cli
 /pkg/cmd/grafana-server/grafana-server
 /pkg/cmd/grafana-server/debug
-/pkg/extensions
```

Comment:
Please revert this change. It's needed even though it mat not look like that.

Reply:
sure thing, but when I was working, that directory kept going missing. not sure why, and so my local dev stopped working.